airbyte-lib/airbyte_lib/executor.py

```diff
@@ -10,11 +10,20 @@
 
 from airbyte_lib.registry import ConnectorMetadata
 
+_LATEST_VERSION = "latest"
+
 
 class Executor(ABC):
-    def __init__(self, metadata: ConnectorMetadata, target_version: str = "latest"):
+    def __init__(
+        self,
+        metadata: ConnectorMetadata,
```

Comment:
Could we handle `ConnectorMetadataDefinitionsV0` models defined [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py#L285) instead of using a custom dataclass?

Reply:
I looked into this briefly when implementing but skipped over it as it wasn't a blocker. In general I agree that we should use the existing definitions, what prompted me to skip is that I doesn't look like this package is published to pypi, so we would can't easily rely on it. Maybe I missed an obvious way here though.

Reply:
Oh yes the fact that it's not published to Pypi is indeed a blocker. If we take the "publish airbyte-lib to pypi via airbyte-ci route" could we also plan to publish the metadata lib to Pypi with the same logic?

Reply:
Sounds great!